package.json

```diff
@@ -60,6 +60,7 @@
   },
   "dependencies": {
     "@nomiclabs/ethereumjs-vm": "^4.2.2",
+    "chai-as-promised": "^7.1.1",
```

Comment:
please use precise versioning!

Reply:
Should be fine to use caret versioning but please move to dev dependencies

Reply:
Moved to dev dependencies and pinned to specific version.